docs/HEOComponents/HEOBackgroundTexture.en.md

```diff
@@ -0,0 +1,8 @@
+# HEOBackgroundTexture
+
+![HEOBackgroundTexture](img/HEOBackgroundTexture.jpg)
+
+This component will display the designated image to the scene background.<br>
+The image will not shift on player movement like Skybox, and position will be constant.
+
+Please make sure that the designated image is formatted to a **1:1** ratio.
```

Comment:
X Please make sure that the designated image is formatted to a **1:1** ratio.
〇 Please make sure that the designated image has a 1:1 aspect ratio.